packages/App/src/store/selectors/userIsLoged.tsx

```diff
@@ -0,0 +1,3 @@
+export const userIsLogIn = (state: any) => {
```

Comment:
Let's add a type for state
`type RootState = ReturnType<typeof combinedReducer>;`